lib/aws/lambda.js

```diff
@@ -158,9 +158,56 @@ function AWSLambda() {
       return;
     }
 
+    self.addParams = function(data) {
+      if(input.custAccount){
+	console.log("set customer creds");
+        input.creds=input.custAccount;
+      }
+	console.log("added permission");
+    }
+
     var lambda = me.preRun(self, input);
     lambda.addPermission(params, me.callback);
   }
+
+   me.assumeRoles = function(input, callback) {
+    console.log("=====started calling lambda function for get credential")
+    input.authorizer_user_guid="b66fdc22-edaa-4710-9ff9-6a77ec834179";
```

Comment:
What guid is this?

Reply:
Right now this value require to auth lambda as parameter but authlambda is not using this value
so we can give any value

Reply:
If no utility cant we send it empty string and add a comment to it